.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: objective-c
+osx_image: xcode61
+rvm:
+- 2.1.2
+install:
+- ./Scripts/install.sh
```

Comment:
What about this:
```
pushd Automata
../Scripts/install.sh
popd
```

I think that would let this work without moving all the files into the repository root. Or did you try that already?